checkin.py

```
@@ -118,6 +118,72 @@ def _extract_user_id_from_session(self, session_cookie: str) -> Optional[str]:
+
+        message = data.get('message')
+        if message is None:
+            message = data.get('msg')
+        if message is None and not isinstance(data.get('data'), (dict, list, type(None))):
+            message = data.get('data')
+        if message is None:
+            message = ''
+        if not isinstance(message, str):
+            try:
+                message = json.dumps(message, ensure_ascii=False)
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)